Contribution-and-Creation/README.md

```diff
@@ -0,0 +1,47 @@
+# OSPO 101 Training Extension Module 6 & 7
+## Open Source Contribution and Creation
+
+This course series is an extension of modules 6 and 7 from the OSPO 101 training course and is based on the [content from the outbound open source guide](https://github.com/todogroup/outbound-oss). Before taking this course, we encourage students to go through the introductory courses on using and creating open source projects:
```

Comment:
I'm not sure if positioning this course as an extension to modules 6 and 7 from the OSPO 101 works best. The modules 6 and 7 are much more focused on practices and operation of concrete open source contributions than at the management of the process. So the audience is different, as the OSPO 101 focused on the people involved in daily contributions, but the outbound guide focus on people managing the framework for doing open source at an organization.

So maybe it would be better to see it the other way around, the outbound guide as a high-level view of managing open source contributions on an organizational level, and the modules from the OSPO 101 as follow-up courses for more details regarding the day-to-day work of doing contributions.

That said, there is some overlap in content, especially in reasons and motivations for open source contributions. Modules 7 from the OSPO 101 has similar content here as also the first section of the outbound guide has.

As a prerequisite for the outbound guide I would see a basic understanding of open source (legal, community, business, technical) and an understanding of managing open source in an organization (for example as OSPO). Not sure, what the best courses are for this. Maybe also something from the OSPO 101? There are some references to material in the outbound guide which could also be seen as prerequisites.